notebooks/12_real_world_applications__emg_sign_classification_in_spd_manifold.ipynb

```diff
@@ -561,88 +561,6 @@
     "warnings.filterwarnings(\"ignore\")"
```

Comment:
<p>This section title should be removed, and the next title Using Euclidean MDM should be brought back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
I just noticed there is a fair amount of work to be done on the notebooks. Let's do them in another PR to keep things atomic.